CONTRIBUTING.md

```diff
@@ -16,13 +15,13 @@ We welcome new feature implementations, patches that fix bugs or code cleanup.
 You can clone from GitHub with:
 
 ```bash
-git clone git@github.com:aquarist-labs/s3gw-tools
+git clone git@github.com:aquarist-labs/s3gw
 ```
 
 Or:
 
 ```bash
-git clone git://github.com/aquarist-labs/s3gw-tools
+git clone git://github.com/aquarist-labs/s3gw
 ```
```

Comment:
Stick with one of the two. This doesn't need to be an in-depth guide on how to use git.